### ISSUE TITLE
Review the Authentication and Authorization - Users field

### ISSUE THREAD

**Issue by amyburness** [iwmihq/miniSASS#1104]
Under 'Authentication and Authorization - Users' in the admin environment, there is still a username field?

**Comment by zamuzakki** [comment]
@amyburness Yes, since the username field is a default field in Django User. Now that we use email to login, it's still there but unused.

**Comment by meomancer** [comment]
We can override form on the django admin to display
Create :
- email, password and confirm_password

And when form is saved, just use email as username

Edit:
- hide username
- move email to the top

**Comment by amyburness** [comment]
@meomancer The old site used user names and the new one uses email addresses. this means that some emails represent 2 or more old users but in the new site each is unique.  Just for context.

For the admin env we should just see the email address.
